labelImgPlusPlus.py

```
@@ -240,10 +564,24 @@ def __init__(self, default_filename=None, default_prefdef_class_file=None, defau
         self.setCentralWidget(scroll)
         self.addDockWidget(Qt.RightDockWidgetArea, self.dock)
         self.addDockWidget(Qt.RightDockWidgetArea, self.file_dock)
-        self.file_dock.setFeatures(QDockWidget.DockWidgetFloatable)
 
+        # Configure dock features - all docks are movable for resizing
+        # DockWidgetMovable enables drag-to-rearrange and proper splitter resizing
+        dock_features_all = (QDockWidget.DockWidgetMovable |
```

Ruff (0.14.14)

[error] Local variable `dock_features_all` is assigned to but never used

Remove assignment to unused variable `dock_features_all`

(F841)